### ISSUE TITLE
Provisional questions

### ISSUE THREAD

**Issue by tengerye** [tengerye/orthogonal-denoising-autoencoder#1]
1. How did the paper apply the trained model in classification? Is the model still suitable for other tasks?
   Answer: please notice the model is not restricted to any specific task, but it is a model of representation learning. In other ways, the model learns effective, robust 'new' representation from feature sets of multiple views. With the new representation, you can apply any algorithm for specific task, e.g., SVM for classification; DBSCAN for clustering.
2. Why my result of Experiment 1 using your code is not as good as reported in the paper?
   Answer: The backpropogation can not guarantee the global optimal, which means the result is different every time. As the parameters in the code are not trivial tuned, the result may vary slightly. You can try different parameters yourself. I will add parameters-tuning for first experiment in further publications. After all, most algorithms can still adapt to these non-champion representations.
3. In your second experiment, why did you not use validation set? And it seems your model is just slightly better than MKL.
   Answer: evaluation reasoning may be included in further publications, but can not fully argue here. The main reason is because the aim of representation learning is learning overall better representation, thus all the values of parameters should be counted (but limited trail is possible). For the second question, please notice the MKL has great advantage considering SVM classifier. Besides, MKL has long history and slightly better is better after all. Further study will surely improve my model.
